config.wyam

```diff
@@ -226,6 +227,12 @@ Pipelines.Add("Post Search Data",
     WriteFiles((doc, ctx) => "data/search/posts.json")
 );
 
+Pipelines.Add("OPML feed export", 
+    ReadFiles("feeds.cshtml"),
+    Razor(),
+    WriteFiles(".opml")
+);

```

Comment:
The `feeds.cshtml` file should probably be named `_feeds.cshtml`, otherwise the "Pages" pipeline will also read it and might overwrite it (or this pipeline would overwrite that one - either way not ideal). The "Pages" pipeline reads all `*.cshtml` files unless they start with an underscore. If you do that, then this pipeline should probably be:
```
Pipelines.Add("OPML feed export", 
    ReadFiles("_feeds.cshtml"),
    Razor(),
    WriteFiles((doc, ctx) => "feeds.opml")
);
```